lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -137,7 +136,7 @@ describe('FilePickerApp', () => {
   it('shows LMS file dialog when "Select PDF from <LMS Name>" is clicked', () => {
```

Comment:
The description here (`<LMS Name>`) looks like it should be just "Canvas".

Reply:
Fixed